deb/openmediavault/var/www/openmediavault/css/theme-triton-rtl.scss

```diff
@@ -0,0 +1,292 @@
+/**
```

Comment:
Please extract only the necessary changes here. This way there is no duplicate code and the changes that are needed for RTL are visible.